index.html

```diff
@@ -2,11 +2,125 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
+    <title>Туристический город Седона</title>
   </head>
   <body>
+    <header class="page-header">
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link" href="sedona.html">Главная</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="about.html">О Седоне</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="catalog.html">Гостиницы</a>
+          </li>
+        </ul>
+        <a class="logo"><img src="" alt="Логотип сайта города Седона"></a>
```

Comment:
Я рекомендую для хорошей читаемость вложенные теги на отдельной строке писать:
```suggestion
        <a class="logo">
          <img src="" alt="Логотип сайта города Седона">
        </a>
```